### ISSUE TITLE
A conflict between Black and Flake8

### ISSUE THREAD

**Issue by DaleSeo** [prodigyeducation/python-graphql-client#14]
**Describe the bug**
 It seems like the way Black formats the code can cause the E231 violation during the Flake8 check.
```
./tests/test_graphql_client.py:88:75: E231 missing whitespace after ','
./tests/test_graphql_client.py:189:75: E231 missing whitespace after ','
```
I tried to manually add a whitespace after `,` at the two places above but Black removes them during pre-commit.

**To Reproduce**
Refer to the following failed jobs on Github Actions.
- https://github.com/prodigyeducation/python-graphql-client/runs/409715041?check_suite_focus=true#step:6:9
- https://github.com/prodigyeducation/python-graphql-client/runs/407818566?check_suite_focus=true#step:6:9
- https://github.com/prodigyeducation/python-graphql-client/runs/407814417?check_suite_focus=true#step:6:9

**Expected behavior**
Falke8 shouldn't complain about the formatted code by Black.

**Additional context**
A simple fix could be to tweaking `.setup.cfg`. so that Falke8 ignores the E231 linting rule.

**Comment by ChrisZhangProdigy** [comment]
@DaleSeo can you assign this to me?

**Comment by steve148** [comment]
Assigned :pray:

**Comment by steve148** [comment]
Ok looking at this further, it seems that the two lines causing flake8 to complain are because of a trailing comma within the dictionary. This isn't something that black automatically adds / removes / edits when doing it's formatting it seems. This linter warning can be fixed simply by removing the trailing comma from within each dictionary.

**Comment by steve148** [comment]
Fixed in #17 so closing this now.
